src/formEditor/FormXmlParser.ts

```
@@ -0,0 +1,421 @@
+  'CommandBar',
+  'CommandBarButton',
+  'Separator',
+  'Navigator',
+  'ContextMenu',
+]);
+
+function parseElements(nodes: XmlNode[]): FormElement[] {
+  const result: FormElement[] = [];
+  for (const node of nodes) {
+    const tagName = getTagName(node);
+    if (!tagName) continue;
```

ESLint

[error] Expected { after 'if' condition.

(curly)